### PR TITLE
Improve book based on feedback

### PR DESCRIPTION
**Pull request by nnatter**
See #587

### CODE REVIEW

File: book/content-architecture.rst [hunk 1]
```diff
@@ -39,4 +39,4 @@ for each country, but the rest of the page should be exactly the same.
 
 .. _PHPCR: http://phpcr.github.io/
 
-With the content architecture on our mind we'll create a webspace.
+With the content architecture on our mind, we can :doc:`webspaces`.
```
Comment (dbu) [inline book/content-architecture.rst:42]:
```suggestion
With the content architecture on our mind, we can create :doc:`webspaces`.
```

**Review by dbu — COMMENTED** [review]
cool!

File: book/content-architecture.rst (hunk 1 above)
Comment (nnatter) [reply book/content-architecture.rst:42]: I think `:doc:webspaces` will inline the title of the document which is `Setup a Webspace` 🙂

Comment (dbu) [reply book/content-architecture.rst:42]: ah if that is correct then ignore me :-D

**Comment by alexander-schranz** [comment]
@dbu thx for the feedback again, this is very welcome and we appreciate it alot.
@nnatter thx for fixing this.

**Pull request merged**